extensions/wikia/ArticleVideo/styles/_video-controls.scss

```diff
@@ -0,0 +1,159 @@
+/**
+ * Styles affecting the video controls that cannot be changed through the
+ * config.json file. Some of these need to be marked as important due to inline
+ * styling and importants already set in the ooyala initial css.
+ *
+ * Some of this is a little hacky. Might be able to cleanup when we pull in the full
+ * html5-skin repo and build our own css.
+ *
+ */
+
+//Vars (Some are from the config.json file)
+$video-player-blue: #2cd3f8;
+$video-scrubber-height: 3px;
+$video-scrubber-played-color: $video-player-blue;
+$video-player-head-color: $video-player-blue;
+$video-player-head-size: 13px;
+$video-player-head-size-scrubbing: 20px;
+$video-player-volume-bar-height: 16px;
+$video-player-volume-bar-off-color: rgba(255,255,255,0.2);
+
+/**
+ * Using SVG data encodings as background images is probably the simplest method to switch out the default ooyala
+ * icons. Whenever you override a control the font should change to transparent to keep the correct
+ * spacing but not show the original icon. But there needs to be a different uri for any color that will need to be used.
+ *
+ * It might make sense to create an svg sprite but I am not even sure if there would be any savings by doing that and it would
+ * be harder to maintain.
```

Comment:
Please add info where are the sources for these icons and how to convert them to the base64 form, so the next person doesn't have to figure it out.